data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/accumulator/SinkAccumulator.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.sink.accumulator;
+
+/**
+ * {@link SinkAccumulator} Accumulate buffer records
+ */
+public interface SinkAccumulator {

```

Comment:
Where is this being used? the inMemoryAccumulator and localFileAccumulator should implement this interface

Reply:
Removed this interface, not being used.